mappings/net/minecraft/block/Block.mapping

```diff
@@ -1,6 +1,6 @@
 CLASS none/akw net/minecraft/block/Block
 	FIELD A stateFactory Lnone/ask;
-	FIELD a DEFAULT_ID Lnone/kp;
+	FIELD a AIR_ID Lnone/kp;
```

Comment:
Ah, I see now. Well, obviously it's also used to register the air block. Should it be named `DEFAULT` just because it's passed into the registry as the default block? It **is** the identifier used for the air block.